Detectors/TPC/calibration/SpacePoints/src/ResidualAggregator.cxx

```diff
@@ -305,6 +314,12 @@ void ResidualsContainer::merge(ResidualsContainer* prev)
   treeOutTrackData = std::move(prev->treeOutTrackData);
   treeOutResidualsUnbinned = std::move(prev->treeOutResidualsUnbinned);
 
+  // since we want to continue using the TTrees of the previous slot, we must
+  // avoid that ROOT deletes them when the TFile of the previous slot is erased
+  treeOutResiduals->SetDirectory(fileOut.get());
```

Comment:
This took me some time to find, since I had not considered the ROOT object ownership which is not automatically transferred with the move above.